Description: Declare byte array as unsigned char rather than char to avoid narrowing
 A step up in pedantics in C++11 has identified a byte array that was not
 declared as unsigned.
Author: William Panlener <wpanlener@gmail.com>
Bug-Debian: https://bugs.debian.org/805160
Forwarded: no

--- a/src/manglerg15.h
+++ b/src/manglerg15.h
@@ -52,7 +52,7 @@
         std::vector<Glib::ustring> events;
 };
 
-const char g15_blank[] = {
+const unsigned char g15_blank[] = {
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
